bmap-rs/src/main.rs

```diff
@@ -111,7 +113,14 @@ fn copy(c: Copy) -> Result<()> {
     }
 
     let mut input = setup_input(&c.image)?;
-    bmap::copy(&mut input, &mut output, &bmap)?;
+    let pb = ProgressBar::new(bmap.total_size());
+    pb.set_style(ProgressStyle::with_template("{spinner:.green} [{elapsed_precise}] [{wide_bar:.cyan/blue}] {bytes}/{total_bytes} ({eta})")
+        .unwrap()
+        .with_key("eta", |state: &ProgressState, w: &mut dyn Write| write!(w, "{:.1}s", state.eta().as_secs_f64()).unwrap())
+        .progress_chars("#>-"));
+    bmap::copy(&mut input, &mut pb.wrap_write(&output), &bmap)?;
```

Comment:
Quick question, does this easily work with the other PR which implements the async case ?

Reply:
there is a async version of the wrap